src/components/shared/DropdownField/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+type DropdownFieldProps = {
+  id: string;
+  label?: string;
+  children: React.ReactNodeArray;
+};
+export const DropdownField = ({ id, label, children }: DropdownFieldProps) => (
```

Comment:
Is there a reason that this doesn't just take a list of values and labels and build the options itself?

Reply:
I personally prefer to pass children in this situation, rather than passing a list of objects as props. It's more declarative.

One of the big advantaged of using `children` is allows engineers to have maximum flexibility. Since these are common components that can be used in different contexts, I opted to have that flexibility.